assets/styles/variables/_colors.scss

```diff
@@ -24,4 +24,5 @@
   @include generate-color-variants(--_c-dark-red-secondary, #801900);
   @include generate-color-variants(--_c-light-red-secondary, #ffb6b6);
   @include generate-color-variants(--_c-blue-secondary, #007bff);
+  @include generate-color-variants(--_c-gray-shadow, rgba(14, 10, 10, 0.5));
```

Comment:
For consistency can we use Hexadecimal codes?

Reply:
Took RGBA color from mockup for box-shadow, Hex code not contains the _opacity_ with _Hex code_, which can be difficult to apply on box-shadow. I kept this in global variable as it can be used anywhere. Please suggest we should move to required page only.